cpp/example_code/s3/get_put_object_acl.cpp

```diff
@@ -284,16 +284,24 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        const Aws::String bucket_name = "my-bucket";
-        const Aws::String object_key = "my-file.txt";
+        //TODO: Change to the name of an actual bucket in your account.
+        const Aws::String bucket_name = "DOC-EXAMPLE-BUCKET";
+        //TODO: Create a file called "my-file.txt" in the local folder where your executables are built to.
+        const Aws::String object_name = "my-file.txt";
+        //TODO: Set to the region of your profile.
+        const Aws::String region = "us-east-1";
 
-        // Set the ACL's owner information. 
+
+        //TODO: Set the ACL's owner information (if it is your bucket, then you want your canonical id). 
```

Comment:
We've already told them it must be their bucket, so just say "Your canonical ID."